src/main/java/eggventory/parsers/Parser.java

```diff
@@ -42,7 +43,7 @@ private Command handleListInput(String listInput) throws BadInputException,
         listInput = listInput.strip(); //Remove leading/trailing whitespace.
 
         //Extract the first word.
-        //inputArr[0] is the main command word.
+        //inputArr[0] i s the main command word.
```

Comment:
can i request that you recommit to fix this lol

Reply:
oh why do i always